### PR TITLE
Add tip about deleting subdirectories

### LINKED ISSUE #76: Delete empty dirs with excluded node_modules
I'm using such code to remove all the files except `node_modules` folders.

```js
del([`${folder}/**/*`, '!**/node_modules/**'], {
    dot: true,
    nodir: true
  })
```

It works fine, but the small problem - it leaves empty directories without files. If set `nodir: false` then it will also deletes `node_modules` despite the negate pattern.

Is there a solution to this problem?

### PR DESCRIPTION
**Pull request by deadcoder0904**
Closes #76

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -44,6 +44,12 @@ You have to explicitly ignore the parent directories too:
 del.sync(['public/assets/**', '!public/assets', '!public/assets/goat.png']);
 ```
 
+To delete all sub-directories inside `public/`, you can do:
```
Comment (sindresorhus) [inline readme.md]: This doesn't look related to the above. It should be somewhere else. It's also not clear from the text whether this deletes all sub-directories *and* files directly under `public/` or just it's sub-directories.

File: readme.md [hunk 2]
```diff
@@ -44,6 +44,12 @@ You have to explicitly ignore the parent directories too:
...
+
+```js
+dev.sync([`public/*/`]);
```
Comment (kevva) [inline readme.md]: `dev` => `del`

File: readme.md (hunk 1 above)
Comment (deadcoder0904) [reply readme.md]: where should I put it? it only deletes all sub-directories inside `public/` (no files are deleted)

File: readme.md (hunk 2 above)
Comment (deadcoder0904) [reply readme.md]: haha done

File: readme.md (hunk 1 above)
Comment (sindresorhus) [reply readme.md]: In a `## Tips` section.

> it only deletes all sub-directories inside public/ (no files are deleted)

That should be made clear in the text.
